python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -1609,14 +1630,17 @@ def run_transformer_int4_fp16_loadlowbit_gpu_win(repo_id,
                 in_out_len = in_out.split("-")
                 in_len = int(in_out_len[0])
                 out_len = int(in_out_len[1])
-                input_str = get_continuation_input_str(in_len, tokenizer)
-                # As different tokenizer has different encodings,
-                # slice the input_ids to ensure the prompt length is required length.
-                input_ids = tokenizer.encode(input_str, return_tensors="pt")
-                input_ids = input_ids[:, :in_len]
-                true_str = tokenizer.batch_decode(input_ids)[0]
-                input_list = [true_str] * batch_size
-                input_ids = tokenizer(input_list, return_tensors="pt").input_ids.to('xpu')
+                if repo_id not in DUMMY_IDS:
+                    input_str = get_continuation_input_str(in_len, tokenizer)
+                    # As different tokenizer has different encodings,
+                    # slice the input_ids to ensure the prompt length is required length.
+                    input_ids = tokenizer.encode(input_str, return_tensors="pt")
+                    input_ids = input_ids[:, :in_len]
+                    true_str = tokenizer.batch_decode(input_ids)[0]
+                    input_list = [true_str] * batch_size
+                    input_ids = tokenizer(input_list, return_tensors="pt").input_ids.to('xpu')
+                else:
+                    input_ids = torch.randint(1000, 2000, [1, in_len], dtype=torch.int64).to('xpu')
```

Comment:
`input_ids = torch.randint(1000, 2000, [batch_size, in_len], dtype=torch.int64).to('xpu')`